EnglishTranslation-master/QUEST_LV_0100.tsv

```diff
@@ -10660,37 +10660,45 @@ QUEST_LV_0100_20151102_010634	If it were anyone else, I would have stopped you b
 QUEST_LV_0100_20151102_010635	I need to rest here for a little bit. I'm running out of breath.{nl}It's dangerous for me to stay any longer with you, because I don't want to risk my life.
 QUEST_LV_0100_20151102_010636	I don't know about the identity of the light bead.{nl}However, it does not have that same feeling the demon-produceds have.

```

Comment:
Is that referring to a beam of light? Or is it literally a light bead? Also demon-produceds? You can say that the demons produce.